tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from "@playwright/test"
+import { randomEmail, randomString } from "./helpers"
+
+const login = async (page: Page) => {
```

Comment:
Just FYI there is a more straightforward way to write this. Reads at least to me more naturally ("aha, this is a function and not a constant variable!"). You can also leave it as is if you feel more comfortable with that syntax.

```suggestion
async function login(page: Page) => {
```